### DISCUSSION TITLE
Get stuck after build production

### DISCUSSION THREAD

**Discussion by tuananh281098** [umijs/umi#8855] [Q&A]
Hi guys. After run yarn build command, I always got this error for some pages. I tried to read the source code and tried to fixed but I haven't done yet. Please help me
![image](https://user-images.githubusercontent.com/28264130/182630864-12a93f34-3493-4420-b2b0-b07bc2bf548c.png)

**Comment by fz6m** [comment]
Due to the lack of enough information, please provide a demo of [Minimal Reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to help resolve the problem.

**Reply by tuananh281098** [reply]
@fz6m 
I tried to remove node_modules then reinstall package and rebuilt. It works well now.
